ast/Trees.cc

```diff
@@ -1002,7 +1002,12 @@ string KeywordArg::toStringWithTabs(const core::GlobalState &gs, int tabs) const
 }
 
 string OptionalArg::toStringWithTabs(const core::GlobalState &gs, int tabs) const {
-    return this->expr->toStringWithTabs(gs, tabs) + " = " + this->default_->toStringWithTabs(gs, tabs);
+    stringstream buf;
+    buf << this->expr->toStringWithTabs(gs, tabs);
+    if (this->default_) {
+        buf << " = " << this->default_->toStringWithTabs(gs, tabs);
```

Comment:
Why not use a `fmt::memory_buffer` here instead?

Reply:
I'm just cargoculting this whole file. Feel free to change them all in the codebase